tests/unit/test_networks.cpp

```diff
@@ -222,6 +276,24 @@ TEST_F(MNISTModel, Forward) {
     validate();
 }
 
+TEST_F(MNISTModel, ForwardTrainFalse) {
+    // Init
+    dw::Tensor input(in.shape());
+    dw::initializer::uniform(input);
+
+    model.train(false);
+    train = false;
```

Comment:
Anyway can't understand what is it ? Why it's needed `train` ? Why don't just `model.train(false)`

Reply:
OK, got it, the ordering is confusing
```
train = false
model.train(train)
```
or make it as a parameter not a class member
But let do it later.